components/lib/datatable/BodyCell.js

```diff
@@ -463,14 +463,16 @@ export const BodyCell = React.memo((props) => {
         }
     }, [props.editingMeta]);
 
-    useUpdateEffect(() => {
+    /* eslint-disable */
```

Comment:
You should not have to disable eslint?

Reply:
I didn't want to disable that, but I couldn't build it successfully, I'm not very familiar with react, please tell me how to do better

```sh
$ npm run build:lib

> primereact@8.7.3-SNAPSHOT build:lib
> NODE_ENV=production INPUT_DIR=components/lib/ OUTPUT_DIR=dist/ npm run build:package


> primereact@8.7.3-SNAPSHOT build:package
> npm run build:check && rollup -c && gulp build-resources && npm run build:api


> primereact@8.7.3-SNAPSHOT build:check
> npm run lint && npm run format:check && npm run type:check && npm run security:check && (NODE_ENV=test npm run test:check)


> primereact@8.7.3-SNAPSHOT lint
> next lint --max-warnings=0


./components/lib/datatable/BodyCell.js
473:8  Warning: React Hook React.useEffect has missing dependencies: 'editingKey', 'getCellCallbackParams', and 'props'. Either include them or remove the dependency array. However, 'props' will change when *any* prop changes, so the preferred fix is to destructure the 'props' object outside of the useEffect call and refer to those specific props inside React.useEffect.  react-hooks/exhaustive-deps
```